### PR TITLE
Add sprint 4 information

### PR DESCRIPTION
**Pull request by MihaiTabara**

### CODE REVIEW

File: sprints/20210129-planning-maintenance-self-serve/sprint-review.md [hunk 1]
```diff
@@ -4,13 +4,15 @@ At the beginning of 2021, our team headcount decreased. With two previous iterat
...
 # Details
+**Product Owner**:
 - geckoview/android-components cleanup: aki
 - automate mac signer maintenance: bhearsum
 - password rotations:
 - pushhealth: sclements
 - interruptduty:
 - greening-up tests: jmaher
 - releng 2021 planning:
+**Scrum Master**: unassigned
```
Comment (MihaiTabara) [inline sprints/20210129-planning-maintenance-self-serve/sprint-review.md]: You've done a lot of heavylifting here, @escapewindow should I put your name down instead?

Comment (escapewindow) [reply sprints/20210129-planning-maintenance-self-serve/sprint-review.md]: Possibly. I'm not really a real scrum master, maybe me+team?

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md [hunk 2]
```diff
@@ -1,13 +1,56 @@
 # Sprint Review - Loose ends and sprint mechanics, February 08, 2021 - February 26, 2021
 
+Sprint 4 added some momentum to our sprint working model. Some seeds we have planted
+in sprin3 were pending sprouting so we dedicated the time for this sprint to
```
Comment (escapewindow) [inline sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md]: * sprint 3

Hm, maybe `s,pending sprouting,unfinished` ?

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md [hunk 3]
```diff
@@ -1,13 +1,56 @@
...
+close some loose ends and put some things behind us in the Done sections.
 
-**Product Owner**: {{PRODUCT_OWNER}}
-**Scrum Master**: {{SCRUM_MASTER}}
+# Details
+**Product Owner**:
+- automate mac signer maintenance: bhearsum
+- activedata eol: mtabara
+- password rotations:
```
Comment (escapewindow) [inline sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md]: might be me

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md [hunk 4]
```diff
@@ -1,13 +1,56 @@
...
+- mdc1: jmaher
+- system-addons balrog: aki
+- releng for mozilla: bhearsum
+- releng hiring: mtabara
+- firefox mr1: mtabara
+**Scrum Master**: unassigned
 
 ## Goal
 
+While working on the 2021 releng roadmap, for sprint 4 we buckled together a couple
```
Comment (escapewindow) [inline sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md]: Hm, I'm thinking `s,buckled,cobbled,` per [definition 3](https://www.merriam-webster.com/dictionary/cobble)

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md [hunk 5]
```diff
@@ -1,13 +1,56 @@
...
+6. [Interruptduty](https://jira.mozilla.com/browse/RELENG-159)
+7. [migrate OSX perf tests from R7 minis (w/Mojave) -> R8 minis (w/Catalina)](https://jira.mozilla.com/browse/RELENG-251)
+8. [bitbar capacity planning](https://jira.mozilla.com/browse/RELENG-258)
+9. [datacenter migration and changes](https://jira.mozilla.com/browse/RELENG-276)
+10. [addons-pipeline: support submitting system addons to balrog](https://jira.mozilla.com/browse/RELENG-291)
+11. [RelEng for Mozilla Planning](https://jira.mozilla.com/browse/RELENG-335)
+12. [RelEng hiring](https://jira.mozilla.com/browse/RELENG-342)
+13. [Firefox MR1](https://jira.mozilla.com/browse/RELENG-347)
 
 ## Outcomes
```
Comment (escapewindow) [inline sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md:55]: We can now fill out these sections, now that we've had the review+retrospective?

**Review by escapewindow — APPROVED** [review]

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md (hunk 4 above)
Comment (MihaiTabara) [reply sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md]: 👍 TIL :)

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md (hunk 5 above)
Comment (MihaiTabara) [reply sprints/20210226-loose-ends-sprint-mechanics/sprint-review.md:55]: Yep, added it now.

**Comment by MihaiTabara** [comment]
@escapewindow mind if you re-review this? just added the review + retro sections

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-retrospective.md [hunk 6]
```diff
@@ -1,11 +1,24 @@
...
 ## What worked
-*
+* We seem to finish 50-some story points per sprint with our reduced team.
+* We find value in the standups because knowing what each other works on gives us possibility to help each other with the interruptduty workload and bus factor
+* We're more and more confortable with using JIRA and its mechanics
+* We're getting better at tracking interruptduty items
 
 ## What didn't
-*
+* We began this sprint with nearly 100 story points, then went to ~115
```
Comment (escapewindow) [inline sprints/20210226-loose-ends-sprint-mechanics/sprint-retrospective.md]: Ended up with 36 unfinished story points

File: sprints/20210226-loose-ends-sprint-mechanics/sprint-retrospective.md [hunk 7]
```diff
@@ -1,11 +1,24 @@
...
 
 ## Process changes for next sprint
-* {{CHANGE}}
-  * **ACTION**:
+* We're questioning whether we should continue adjusting items to the sprint or stay focused to the initial planning and drop anything else
+* Wondering whether stricter sorting into “MVP” and “stretch goal” buckets could help with prioritizing items during the sprint
+* Considering to avoid adding interruptduty items to the sprint, but add them to jira; there's a report for "non-sprint issues resolved during the sprint"
+* Added “Blocked” column in the JIRA sprint for future iterations
+* For sprint 5, we create a list of items, out of which we define the realistic set of the ones we think we can do so that we can make a better estimate
+* Anything over a 1.5h maybe interruptdy worthy-ticket
```
Comment (escapewindow) [inline sprints/20210226-loose-ends-sprint-mechanics/sprint-retrospective.md]: *interruptduty

**Review by escapewindow — APPROVED** [review]

**Pull request merged**